test/fixtures/files/label.json

```diff
@@ -0,0 +1,132 @@
+{
```

Comment:
This file is a direct copy from `test/fixtures/github_webhooks/label.json`. I just copied it so the `load_fixture` method would work while I was writing the tests, but we can use the original file if you like, or pass in a label stub to slim things down.